addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3482,7 +3482,10 @@ msgctxt "#800"
 msgid "Music library needs to rescan tags from files. Would you like to scan now?"
 msgstr ""
 
-#empty string with id 801
+#: xbmc/music/windows/GUIWindowMusicBase.cpp
+msgctxt "#801"
+msgid "Fetch additional information for albums and artists?[CR]This could take some time so you may prefer to do this later"
```

Comment:
OK, any band allocated to anything specific?  Or just pick the next number at the end?
I just put the new message next to the related old one because there was a gap.  Dotting text about seems really confusing to me